mailprocessing/mail/maildir.py

```diff
@@ -53,8 +53,8 @@ def copy(self, maildir, **kwargs):
         folder_list = self._processor.path_ensure_prefix(maildir, sep=sep)
         maildir = self._processor.list_path(folder_list, sep=sep)
 
-        self._processor.log("==> Copying to {0}".format(target))
-        self._copy(maildir, *kwargs)
+        self._processor.log("==> Copying to {0}".format(maildir))
```

Comment:
`target` is an undefined variable, but `maildir` exists here and seems like what we meant.